gamification/src/main/java/juja/microservices/gamification/user/ServiceImpl.java

```diff
@@ -46,7 +44,7 @@
 public class ServiceImpl implements Service {
```

Comment:
@sLisnychyi Service is tooooo abstract name as for class as for interface - need to change to something more concrete (if will decide to leave it)